### ISSUE TITLE
[Feature Request] Modifying variable's value

### ISSUE THREAD

**Issue by royken** [creativetimofficial/argon-dashboard-angular#8]
### What is your enhancement?
Hi,

How and where can I modify default variable values (color, font, ...)

**Comment by chelaruc** [comment]
Hi, @royken. Thank you for using our product. Our variables are in the file `src/assets/scss/custom/_variables.scss`.

**Comment by royken** [comment]
hello @chelaruc  Thank you for your response.
By the way, is it possible to create a custom layout?

**Comment by chelaruc** [comment]
Yes, it is possible. You can take an example from our layouts(admin and auth) from the `src/app/layouts` folder.
